fre/make/runFremake.py

```diff
@@ -187,26 +190,32 @@ def fremake_run(yamlfile,platform,target,parallel,jobs,no_parallel_checkout,verb
 
                 dockerBuild.writeRunscript(RUNenv,containerRun,tmpDir+"/execrunscript.sh")
 
-                ## Run the dockerfile; build the container
-                dockerBuild.build(containerBuild,containerRun)
+                ## Run the dockerfile and create the container if execute option is given
+                ## otherwise create a build script and print out its path
+                if execute:
```

Comment:
These steps would also need to be added in `createDocker.py` as well

Reply:
Oh wait, to clarify, `createDocker.py` already has the execute (run) or not options. If execute, it'll created the dockerfile and build. If not, it'll just create the dockerfile. So it would need the ability to create the buildScript.

Reply:
But maybe that can be done in another PR since this is mainly for run-fremake

Reply:
Sounds good, I can put those changes in another PR.